src/cook-web/src/components/billing/BillingPlanComparisonTable.module.scss

```
@@ -0,0 +1,277 @@
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.featureColumnItem {
+  min-width: 0;
+}
+
+.featureColumnItemText {
+  color: $color-fg;
+  word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)